app/views/admin/_certificate.erb

```diff
@@ -0,0 +1,63 @@
+<% certificates.reverse.each do |certificate| %>
+  <table class="govuk-table">
+    <caption class="govuk-table__caption" id="certificates/<%= certificate.id %>">Certificate ID: <%= certificate.id %></caption>
+    <tbody class="govuk-table__body">
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Name</th>
+        <td class="govuk-table__cell"><%= certificate.x509.subject.to_s %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Enabled</th>
+        <td class="govuk-table__cell"><%= certificate.enabled %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Usage</th>
+        <td class="govuk-table__cell"><%= certificate.usage %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Valid from</th>
+        <td class="govuk-table__cell"><%= certificate.x509.not_before %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Valid until</th>
+        <td class="govuk-table__cell"><%= certificate.x509.not_after %></td>
+      </tr>
+      <tr class="govuk-table__row">
+        <th scope="row" class="govuk-table__header">Certificate</th>
+        <td class="govuk-table__cell">
+          <details class="govuk-details" data-module="govuk-details">
+            <summary class="govuk-details__summary">
+              <span class="govuk-details__summary-text">
+                Show certificate
+              </span>
+            </summary>
+            <div class="govuk-details__text">
+              <pre><%= certificate.x509.to_text %></pre>
+              <pre style="overflow-wrap:anywhere;white-space:normal"><%= certificate.value %></pre>
```

Comment:
Is it appropriate to have styling on the view?

Reply:
ah well spotted.. I was going to fix it and then forgot!

Reply:
fixed